index.html

```diff
@@ -2448,6 +2448,128 @@ <h2>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Launch queue and launch params
+      </h2>
+      <p>
+        TODO: This should move to its own spec.
+      </p>
+      <section data-dfn-for="LaunchParams">
+        <h3>
+          <dfn>LaunchParams</dfn> interface
+        </h3>
+        <pre class="idl">
+          [Exposed=Window] interface LaunchParams {
+            readonly attribute DOMString? targetURL;
+          };
+        </pre>
+        <section>
+          <h2>
+            <dfn>targetURL</dfn> attribute
+          </h2>
+          <p>
+            The {{LaunchParams/targetURL}} attribute is a [=string=] that 
+            represents the [=URL=] of the launch [=manifest/within scope=] 
+            of the application. This attribute is set by the 
+            [=Handling Web App Launches=] process.
+          </p>
+        </section>
+      </section>
+      <section data-dfn-for="LaunchConsumer">
+        <h3>
+          <dfn>LaunchConsumer</dfn> function
+        </h3>
+        <pre class="idl">
+          callback LaunchConsumer = any (LaunchParams params);
+        </pre>
+      </section>
+      <section data-dfn-for="LaunchQueue">
+        <h3>
+          <dfn>LaunchQueue</dfn> interface
+        </h3>
+        <pre class="idl">
+          partial interface Window {
+            readonly attribute LaunchQueue launchQueue;
+          };
+
+          [Exposed=Window] interface LaunchQueue {
+            undefined setConsumer(LaunchConsumer consumer);
+          };
+        </pre>
+        <p>
+          {{LaunchQueue}} has an <dfn>unconsumed launch params</dfn> which 
```

Comment:
I don't follow these two statements. Is this supposed to be a description of the two states `LaunchQueue` can be in? Do we need to add verbiage around that?

Reply:
The spec parser is saying these a duplicate definitions of what's above
> A [window](https://html.spec.whatwg.org/multipage/window-object.html#dom-window) has an associated [LaunchConsumer](https://github.com/dmurph/manifest/pull/1/files#dom-launchconsumer) assigned launch consumer.
> A [window](https://html.spec.whatwg.org/multipage/window-object.html#dom-window) has an associated [LaunchParams](https://github.com/dmurph/manifest/pull/1/files#dom-launchparams) unconsumed launch params.

The above is probably fine to keep.

Reply:
Should I remove one of the definitions? If the launch queue and launch params section will eventually move to its own spec then maybe its okay that it's duplicated?

Reply:
Done